pkg/defaults/defaults.go

```diff
@@ -21,6 +21,11 @@ var (
 	LocalArtifactMirrorImage = ""
 )
 
+// Holds the default no proxy values.
+var (
+	DefaultNoProxy = []string{"localhost", "127.0.0.1", ".default", ".local", ".svc", "kubernetes", "kotsadm-rqlite", "10.0.0.0/8"}
```

Comment:
Including `10.0.0.0/8` in this with no way to remove that is going to cause problems

Ideally we would include the actual pod and service CIDRs in this instead